Makefile.am

```diff
@@ -142,7 +142,7 @@ linux_platform_headers = \
 	zfs/ZfsCompressedArcMeter.h
 
 if HTOP_LINUX
-AM_LDFLAGS += -rdynamic
+AM_LDFLAGS += -rdynamic -lrt
```

Comment:
Shouldn't this be detected via `configure.ac` instead? Similar to what is done for `dlopen` …

Reply:
Probably, I'll give it a try when I'm back in front of a computer